com.paledega.alpinelinux.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.paledega.alpinelinux
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: alpine
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission.

Reply:
filesystem permission removed. 

Other permissions are required and vital.